lib/rsgem/tasks/base.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module RSGem
+  module Tasks
+    #
+    # Base class for task objects.
+    # Child classes must implement the instance method +perform+.
+    #
+    class Base
+      attr_reader :context, :extras
+
+      def initialize(context:, **extras)
+        @context = context
+        @extras = extras
```

Comment:
can you explain a bit about why you decided to name it `extras` , it sounds a bit awkward to me this name at first sight

Reply:
Any suggestions? It's just a set of optional arguments

Reply:
args ?

Reply:
Done